src/global.css

```diff
@@ -2,4 +2,14 @@
     text-align: center;
     background-color: gainsboro;
     padding: 10px;
+}
+
+.site-header {
+    background-color: hsl(194, 100%, 66%);
+    text-indent: 20px;
+}
+
+#site-header-heading {
```

Comment:
Try to always use classes :)

Reply:
Ok no problem

Reply:
Basically you can always start off using a class.

The case for using ID is when you absolutely cannot do without. When it's something really unique in the document. 

That you for some reason cannot target using just a class. Like a special case for targetting using JavaScript `document.getElementById`. But honestly it's hard to find that case. 

 You can think of the header as something generic that will appear in every page, so not really unique. 

You're safe always using class.

Reply:
Ahh thank you for the explanation!